barman/compression.py

```diff
@@ -972,3 +1026,70 @@ def get_file_content(self, filename, archive):
                 )
         else:
             return out
+
+
+class NoneCompression(Compression):
+    name = "none"
+    file_extension = "tar"
+
+    def __init__(self, command):
+        """
+
+        :param command: barman.fs.UnixLocalCommand
+        """
+        self.command = command
+
+    def uncompress(self, src, dst, exclude=None, include_args=None):
```

Comment:
not sure about this method's name, looks like something that decompress something not that leave stuff not compressed

Reply:
That is the implementation of the abstract method inherited from `Compression.uncompress`. It's purpose is to uncompress and extract files from an archive.

There is two way of seeing this, and both make sense to me:
- Maybe a more accurate definition of the function is `extract`(because it represents the final state wanted) . it may or may not imply decompression depending on the compression algorithm -> this implies to rename all uncompress methods with extract 
- Or we can see uncompress for none compression as a special king of compression that does not compress at all. -> in this case nothing to change.

Reply:
sorry my fault i got confused following the various methods names around 😄